utils.php

```diff
@@ -160,7 +160,7 @@ function format_userids($names, $unknown_attendees) {
         }
     } else {
         foreach($names as $i => $name) {
-            $names[$i] = "<@$name[userid]>";
+            $names[$i] = "<@$name>";
```

Comment:
(This comment should go on L154 but since github considers it's not part of the review, it forbids me to comment on that other line)

What about renaming the parameters `names` and `name` to `userids` and `userid` (for consistency with the rest of the code)?

Reply:
Thumb up :) b7e6ba37616841d6bf60c8fe0e57c5294d6f37f6